### PR TITLE
Add dependency injection and move Models directory

### PR DESCRIPTION
**Pull request by kwunkiuma**
The Discord bot is now able to use methods provided by EventSheetService.

Creating an event now calls the respective method in the service, although the method itself does not current have any function.

### CODE REVIEW

File: DiscordBot.DataAccess/DiscordBot.DataAccess.csproj [hunk 1]
```diff
@@ -1,16 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
+    <OutputType>Library</OutputType>
```
Comment (FridaTveit) [inline DiscordBot.DataAccess/DiscordBot.DataAccess.csproj:4]: Do you need this? 🙂

File: DiscordBot/Program.cs [hunk 2]
```diff
@@ -24,9 +25,18 @@ private static async Task MainAsync()
                 LogLevel = LogLevel.Debug
             });
 
+            DependencyCollection dependencies;
+
+            using (var builder = new DependencyCollectionBuilder())
```
Comment (FridaTveit) [inline DiscordBot/Program.cs:30]: No block necessary for `using` with C# 8

File: DiscordBot/Program.cs [hunk 3]
```diff
@@ -24,9 +25,18 @@ private static async Task MainAsync()
                 LogLevel = LogLevel.Debug
             });
 
+            DependencyCollection dependencies;
```
Comment (FridaTveit) [inline DiscordBot/Program.cs:28]: This should be declared when you're assigning to it. Avoiding splitting up declarations and assignments whenever possible! 🙂

**Review by FridaTveit — COMMENTED** [review]
A few comments, otherwise looks good 🙂

File: DiscordBot.DataAccess/DiscordBot.DataAccess.csproj (hunk 1 above)
Comment (kwunkiuma) [reply DiscordBot.DataAccess/DiscordBot.DataAccess.csproj:4]: Seems to work without, will remove.

File: DiscordBot/Program.cs [hunk 4]
```diff
@@ -25,13 +25,10 @@ private static async Task MainAsync()
...
-
-            using (var builder = new DependencyCollectionBuilder())
-            {
-                builder.AddInstance<IEventsSheetsService>(new EventsSheetsService());
-                dependencies = builder.Build();
-            }
+            using var builder = new DependencyCollectionBuilder();
+            builder.AddInstance<IEventsSheetsService>(new EventsSheetsService());
+            var dependencies = builder.Build();
+
```
Comment (FridaTveit) [inline DiscordBot/Program.cs:31]: Unneeded empty line 🙂

**Review by FridaTveit — COMMENTED** [review]
One tiny thing otherwise looks good! 🙂

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Pull request merged**